health_monitor/src/driver_manager.cpp

```diff
@@ -246,38 +246,38 @@ namespace health_monitor
             } 
             else if(are_critical_drivers_operational_car(time_now)=="s_1_l_1_g_0")
             {
-                alert.description = "GPS stopped working but SSC and LIDAR  is still running";
+                alert.description = "GPS Failed";
                 alert.type = cav_msgs::SystemAlert::CAUTION;
                 return alert; 
             }
             else if(are_critical_drivers_operational_car(time_now)=="s_1_l_0_g_1")
             {
-                alert.description = "LIDAR stopped working but SSC and GPS is still running";
+                alert.description = "LIDAR Failed";
                 alert.type = cav_msgs::SystemAlert::WARNING;
                 return alert; 
             }
             else if(are_critical_drivers_operational_car(time_now)=="s_1_l_0_g_0")
             {
-                alert.description = "LIDAR, GPS stopped working but SSC is still running";
+                alert.description = "LIDAR, GPS Failed";
                 alert.type = cav_msgs::SystemAlert::FATAL;
                 return alert; 
             }
             else if(are_critical_drivers_operational_car(time_now)=="s_0")
             {
-                alert.description = "SSC stopped working";
+                alert.description = "SSC Failed";
                 alert.type = cav_msgs::SystemAlert::FATAL;
                 return alert; 
             }
             else
             {
-                alert.description = "UNKNOWN";
+                alert.description = "Missing unknown essential driver";
                 alert.type = cav_msgs::SystemAlert::FATAL;
                 return alert;  
             }
         }
         else
         {
-            alert.description = "UNKNOWN";
+            alert.description = "Wrong params";
```

Comment:
This message could be more descriptive. In a long log file, it would be very ambiguous. How about "Wrong params: need to set either truck, car or dev flag."